packages/flutter_tools/lib/executable.dart

```diff
@@ -127,7 +152,7 @@ Future<Null> main(List<String> args) async {
     return Chain.capture<Future<Null>>(() async {
       await runner.run(args);
       await _exit(0);
-    }, onError: (dynamic error, Chain chain) {
+    }, onError: (dynamic error, Chain chain) async {
```

Comment:
The signature of onError is `void`, which implies it doesn't support async methods.  How about `sendReport(...).then()` instead?